NewForce-Cohort-8/tabloidfullstack-suplex-syndicate
# Cb view my posts ticket4

Description: 
Trello ticket 4: https://trello.com/c/ShqDlByr/4-4-view-current-users-posts
This ticket allows the logged in user to to view only their own posts under "My Posts":

Given the user is in the Tabloid application
When they select the My Posts menu option
Then they should be directed to the "My Posts" list page
And the page should display ALL the Posts authored by the logged-in user
And each post in the list should display the title, author and category
And the list should be in order of creation date with the most recent on top



Testing Instructions:

1. git add and git commit whatever you're working on on your branch.
2. git checkout main.
3. git fetch -a.
4. git checkout my branch: cb-viewMyPosts-ticket4.
5. If you don't see the most recent version of my files, git pull origin cb-viewMyPosts-ticket4.
6. (if not already done) Please open the TabloidFullStack.sln with Visual Studio 2022.
7. launch the API by clicking the execute with debugger solid green button in Visual Studio 2022.
8. (if not already open)Open a new GitBash window and CD into the tabloid file inside the client folder of the project and open Visual Studio Code with the command "code ". Then launch the react app with the command "npm start".
9. In the react app in the browser, if you are not already logged in, you should be directed to the login page. Please log in with this email address: foo@bar.com. This should redirect you back to the home page.
10. On nav bar, click "My Posts". You should be directed to the path "http://localhost:3000/my-posts", and see all the posts for that user. Please verify that title, author and category display for the posts, and that they are ordered with the most recent on top.
11. If your database doesn't contain a post without a publish datetime for this user, please create a new test post under "New Post" from the menu and do not enter a publish datetime in the post. Then click on "My Posts" again and verify that the unpublished post is displaying in the list.

Checklist:
